packages/next/lib/verifyTypeScriptSetup.ts

```diff
@@ -100,7 +100,7 @@ async function checkDependencies({
 export async function verifyTypeScriptSetup(
   dir: string,
   pagesDir: string
-): Promise<void> {
+): Promise<boolean> {
```

Comment:
Is this change needed? It looks like you ended up using `hasTypeScript`.

Reply:
It's being used for `next-dev-server`, in that case the record is sent after using `verifyTypeScriptSetup`, not before like in the case of `build`, so there wasn't a need to manually check for TypeScript again.

Reply:
Now it's also being used by `build`, it detects TypeScript in the way we detect it, so it's quite accurate, and we use the boolean returned by it without having to do any other task.